test/fixtures/rules/std_rule_cases/L016.yml

```diff
@@ -51,6 +51,22 @@ test_fail_line_too_long_with_comments_4:
     rules:
       max_line_length: 80
 
+test_fail_line_too_long_with_comments_ignore_comment_lines:
+  # In this case, the inline comment is NOT on a line by itself (note the
+  # leading comma), but even if we move it onto a line by itself, it's still
+  # too long. In this case, the rule should do nothing, otherwise it triggers
+  # an endless cycle of "fixes" that simply keeps adding blank lines.
+  pass_str: |
+    SELECT
+    c1
+    ,--  the "y variable" and uses_small_subject_line to be the "x variable" in terms of the regression line.
+    c2
+  configs:
+    rules:
+      max_line_length: 80
+      L016:
+        ignore_comment_lines: True
+
```

Comment:
@barrywhart we needed to add this test from your recently merged change to get coverage back up to 100%